src/pages/lib-v1/predictions/transcribe/index.mdx

```diff
@@ -6,7 +6,7 @@ import { INTEGRATION_FILTER_OPTIONS } from '@/utils/filter-data.ts';
   directoryPath="/ChooseFilterPage"
   address="/lib-v1/predictions/transcribe"
   filterKind="platform"
-  filters={['android', 'ios']}
+  filters={['android', 'ios', 'js', 'react-native']}
```

Comment:
I think this one might actually only be supported on iOS

Reply:
It's definitely supported in web and was listed as supported for v5, although it won't work because of its use of buffer so I'm totally down to remove that. Just wanted to keep v5 as is but in this case it makes sense to remove until we can actually fix it for RN